padertorch/ops/mappings.py

```diff
@@ -5,11 +5,34 @@
 
 __all__ = [
     'ACTIVATION_FN_MAP',
-    'POOLING_FN_MAP',
-    'DTYPE_MAP',
 ]
 
-ACTIVATION_FN_MAP = Dispatcher(
+class CallableDispatcher(Dispatcher):
```

Comment:
Would it be better to move this to `paderbox.utils.mapping`, next to `Dispatcher`?

Reply:
I can do that, however @boeddeker wants to make it privat which would not realy make sense in that case.

Reply:
Yes, I agree that making it private and at the same time moving it doesn't make sense. We have to discuss if there are other use-cases outside of `pt.ops.mappings`

Reply:
And we also need a good name for it.
I don't like the current one. But for a private one, it would be ok.